src/global_identifiability.jl

```diff
@@ -176,7 +205,9 @@ function check_identifiability(
             break
         end
     end
-    states_needed = states_needed || isempty(ode.parameters)
+    if !states_needed && isempty(ode.parameters)
+        return [true for _ in funcs_to_check]
```

Comment:
Isn't this vector always empty?

Reply:
In a meaningful situation, no, but technically one can write things like `assess_identifiability(ode, funcs_to_check=[one(ode.poly_ring)])`